server/src/module/settings/settings.service.ts

```diff
@@ -9,11 +9,12 @@ import { StaticSettings } from './settings.static';
 
 @Injectable()
 export class SettingsService extends StaticSettings implements OnApplicationBootstrap {
-    private readonly repository: EntityRepository<Setting>;
-
-    constructor(entityManager: EntityManager) {
+    constructor(
+        private readonly orm: MikroORM,
```

Comment:
Is the `orm` parameter still needed?

Reply:
yes, as without it, `@UseRequestContext()` does not work